packages/core/src/routes-me/verification-code.ts

```diff
@@ -45,7 +45,7 @@ export default function verificationCodeRoutes<T extends AuthedMeRouter>(
       body: object({
         email: string().regex(emailRegEx),
         verificationCode: string().min(1),
-        action: union([literal('changeEmail'), literal('changePassword')]),
+        action: union([literal('changeEmail'), literal('changePassword')]).optional(),
```

Comment:
Per offline discussion, make it a required field.

Reply:
Updatd, ptal